pyvrp/cli.py

```diff
@@ -87,6 +88,9 @@ def solve(
     max_iterations
         Maximum number of iterations for solving. Either ``max_runtime`` or
         ``max_iterations`` must be specified.
+    stats_dir, optional
+        The directory to write runtime statistics to. Enables statistics
+        collection when passed.
```

Comment:
Suggestion: could we also add a `solutions_dir` to save best solutions in VRPLIB format?

Reply:
Yeah, I'll add that.